src/pages/publication/ReferencesPanel.tsx

```diff
@@ -58,27 +67,28 @@ export const ReferencesPanel: React.FC<ReferencesPanelProps> = ({ goToNextTab, s
   }, [setAllFieldsTouched]);
 
   return (
-    <TabPanel
-      ariaLabel="references"
-      goToNextTab={goToNextTab}
-      heading={t('publication:heading.references')}
-      onClickSave={() => savePublication()}>
-      <Field
-        name={ReferenceFieldNames.REFERENCE_TYPE}
-        aria-label="referenceType"
-        variant="outlined"
-        fullWidth
-        component={Select}>
-        {referenceTypeList.map(type => (
-          <MenuItem value={type.value} key={type.value} data-testid={`referenceType-${type}`}>
-            {t(type.label)}
-          </MenuItem>
-        ))}
-      </Field>
+    <TabPanel ariaLabel="references" goToNextTab={goToNextTab} onClickSave={() => savePublication()}>
+      <StyledSelectContainer>
+        <Field name={ReferenceFieldNames.REFERENCE_TYPE}>
+          {({ field: { onChange, name, value } }: any) => (
+            <FormControl variant="outlined" fullWidth>
+              <InputLabel>{t('common:type')}</InputLabel>
+              <Select value={value} onChange={onChange(name, value)}>
+                {Object.keys(referenceTypeLanguageKeyMap).map(type => (
```

Comment:
siden vi trenger både key og value kunne vi forenklet med:
```{Object.entries(referenceTypeLanguageKeyMap).map([key, value] => (...)}```

Reply:
hvorfor er dette bedre?

Reply:
slipper to oppslag i `referenceTypeLanguageKeyMap`. Når vi vet vi trenger både key og value kan vi like godt hente ut begge samtidig IMO

Reply:
jeg skjønner ikke hvordan dette kan fungere. map tar jo inn en string?

Reply:
åja entries ja